src/common/lambda-function.ts

```diff
@@ -43,8 +49,8 @@ export class LambdaFunction {
   public validateVPCDiscovery (funcVPCDiscovery: FuncVPCDiscovery): boolean {
```

Comment:
Since we renamed the other one
```suggestion
  public validateFuncVPCDiscoveryConfig (funcVPCDiscovery: FuncVPCDiscovery): boolean {
```